community/kernel/src/main/java/org/neo4j/kernel/api/txstate/TxStateVisitor.java

```diff
@@ -38,301 +38,240 @@
  */
 public interface TxStateVisitor
 {
-    void visitCreatedNode( long id );
+    default void visitCreatedNode( long id ) {}
```

Comment:
Oh OK, you have a point there... I'll do the normal Adapter/Delegator thing instead.